src/vptstools/s3.py

```diff
@@ -124,7 +124,7 @@ def handle_manifest(manifest_url, bucket, look_back="2day"):
 S3Obj = namedtuple('S3Obj', ['key', 'mtime', 'size', 'ETag'])
 
 
-def s3list(bucket, path, start=None, end=None, recursive=True, list_dirs=True,
+def s3list(bucket, path, start=None, end=None, recursive=True, list_dirs=True,  # noqa: C901
```

Comment:
?

Reply:
This is a temporary fix to overcome the flake8 automatic code checks (see https://www.flake8rules.com/rules/C901.html). Note, the `s3list` function will be removed in the h5_to_csv branch when the inventory setup is fully operational.